api/base/filters.py

```diff
@@ -171,12 +174,13 @@ def bulk_get_values(self, value, field):
         return values
 
     def parse_query_params(self, query_params):
-        """Maps query params to a dict useable for filtering
+        """Maps query params to a dict usable for filtering
```

Comment:
In this case, the spell checker default configuration prefers the change. I like to keep tooling warnings to a minimum so that the warnings I do so are worth the effort.